dashboard/run_data_collection.py

```diff
@@ -115,6 +121,12 @@ def collect_data(self, hpc: const.HPC):
                         None,
                     ),
                 )
+        # get quota
+        quota_output = self.run_cmd(
+            hpc.value, "nn_check_quota | grep 'nesi00213'", timeout=60
```

Comment:
This cmd should be fast, so leave the timeout to its default value unless it starts timing out.